README.md

```diff
@@ -86,17 +86,25 @@ $ ./configure
 $ make dist
 $ ls -l
 ```
-
-You should see a tar file named fourier-0.4.3.tar.gz in the directory. Ccopy fourier-0.4.3.tar.gz to your release webpage.
-
+You should see a tar file named gimp-fourier-plugin-0.4.4.tar.gz in the same directory.
+To verify that the dist package contains all files and nothing is missing, test build it....
+```
+$ tar -xzf gimp-fourier-plugin-0.4.4.tar.gz
+$ cd gimp-fourier-plugin-0.4.4
+$ ./configure --bindir=/usr/lib/gimp/2.0/plug-ins
+$ make
+$ sudo make install
+```
+If no errors, then copy gimp-fourier-plugin-0.4.4.tar.gz to your release webpage.
+NOTE: rpm spec file Source0 URL links to this file.
 
 ## History
 
 ```
 *  v0.4.4 (Aug 2022): 
     - Replaced deprecated functions
-    - Autotools toolchain and spec file by JoesCat
-    - Github action workflow to build plugin
+    - Autotools toolchain and initial_rpm.spec file by Joe Da Silva
```

Comment:
There is also another mention of your name in "Many thanks" sentence at the end of Changelog if you want to change it too.

Reply:
LOL - never do things late at night - missed them.